patches/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -158,15 +158,23 @@
      public ItemEnchantments getEnchantments() {
          return this.getOrDefault(DataComponents.ENCHANTMENTS, ItemEnchantments.EMPTY);
      }
-@@ -933,6 +_,8 @@
+@@ -933,12 +_,16 @@
      }
  
      public void forEachModifier(EquipmentSlot p_332001_, BiConsumer<Holder<Attribute>, AttributeModifier> p_330882_) {
++        // Neo: Use ItemStack extension method, which fires an event for mod-added attributes modifiers
 +        this.getAttributeModifiers(p_332001_).forEach(p_330882_);
-+        if (true) return;
++        if (false) {
```

Comment:
Thoughts on indenting the whole block to make sure that we won't miss changes here when updating?

Reply:
My personal preference when writing patches is to keep them small, contained, and commented, and indenting the block would make a lot of visual noise when looking at the patch itself.

It would, however, be effective I believe in ensuring changes to the block are always noticed during porting.

I'll do the change in my PR, but whoever works on the item attribute modifiers fixes next (as I described in my PR description) will need to do the same for the other `forEachModifier` method.